### DISCUSSION TITLE
Can anyone help me with how to use ResponseBody "isJOSN" under "Tests" tab of a request?

### DISCUSSION THREAD

**Discussion by ajaydwarkani** [thunderclient/thunder-client-support#873] [Q&A] [answered]
Here is the screenshot. What should I put in place of '???'

![image](https://user-images.githubusercontent.com/4305382/201435462-8037ff26-948a-41e8-9c06-1077d09f4772.png)

**Comment by rangav** [comment, marked as answer]
You can put true or false

**Reply by ajaydwarkani** [reply]
Thanks @rangav, I figured too 2 minutes ago ;)
